src/status_im/contexts/settings/wallet/keypairs_and_accounts/actions/view.cljs

```diff
@@ -19,20 +19,30 @@
   [data]
   (rf/dispatch [:open-modal :screen/settings.encrypted-key-pair-qr data]))
 
+(defn on-import-seed-phrase
+  [data]
+  (rf/dispatch [:open-modal :screen/settings.import-seed-phrase data]))
+
 (defn view
-  [props key-pair]
+  [{key-pair :keypair
+    :keys [drawer-props missing-keypair?]}]
```

Comment:
Should we name things `key-pair` or `keypair`? I see other places in the code write things as `keypair`, but I see that the correct spelling would likely be `key-pair`. Thoughts?

Reply:
It's been created as `keypair`.

I recently worked on key pairs and and it was only one use of the `key-pair` variant, so I decided to rename it to `keypair` to make it consistent, otherwise when developers try to read/store in re-frame or pass the keyword as parameter, they'll be confused of what is the variant used... and I'm pretty sure this will cause bugs :sweat_smile: .

At this moment, I think most of the code is using `keypair`, so better to keep it and probably consider to rename it in the future.

tagging @J-Son89  just for visibility.

Reply:
@mohsen-ghafouri Is it okay if I change `key-pair` to `keypair` in the related code? Or is there another reason for naming it `key-pair`?

Reply:
sorry didn't see this message, yeah as we already agreed, will rename all before removing feature flag.

Reply:
Hey guys, let's check what is the most common usage.

I feel bad about reading `keypair` instead of `key-pair`. Maybe we could open a quick PR to change them to the correct name :thinking:  wdyt?

Reply:
@ulisesmac After doing a quick search I think `keypair` might be the most common in the status-mobile codebase, it seems `key-pair` is mostly used for **i18n** labels. Plus `keypair` is used in the status-go code too, so if we wanted it to be consistent between the go code and mobile we'd have to change both.

Reply:
@ulisesmac - It's supposed to be called `Key pair` grammatically but the word `Keypair` is used extensively across both `status-desktop` and `status-go`. It's not a trivial item at this moment, we can make separate PR to rename if required.

Reply:
OK, thanks for providing context guys, let's use it as `keypair` for consistency then :+1: